src/views/credits/credits.scss

```diff
@@ -1,39 +1,205 @@
 @import "../../colors";
 
-.credits {
-    p {
-        line-height: 1.5rem;
-
-        a {
-            white-space: normal;
-            word-wrap: break-word; /* Overrides: https://github.com/LLK/scratch-www/blob/develop/src/main.scss#L43-L47 */
-        }
-    }
+#view {
+    padding: 0;
+}
 
+.credits {
     ul {
         display: flex;
-        margin: 0;
-        padding: 0;
+        margin: 0 auto;
+        padding: 64px 0;
+        max-width: 864px;
         list-style: none;
         flex-wrap: wrap;
+        flex-flow: row wrap;
+        justify-content: center;
+    }
+
+    .content {
+        padding-top: 40px;
     }
 
     li {
         display: inline-block;
-        margin: 10px 0;
-        width: 188px;
-        text-align: center; 
+        margin: 0;
+        padding-bottom: 32px;
+        text-align: center;
+        line-height: 1.25rem;
 
         img {
-            margin: 10px 23px;
-            border: 2px;
-            border-style: solid;
-            border-radius: 50%;
-            border-color: $ui-dark-gray;
+            $img-border: rgba(0, 0, 0, .05);
+            border: 2px solid $img-border;
+            border-radius: 8px;
             background-color: $ui-white;
-            padding: 20px;
-            width: 85px;
-            height: 85px;
+            width: 80px;
+            height: 80px;
+        }
+
+        span {
```

Comment:
Styling spans is not a great idea here for two reasons:
1. You are using a span in a specific way (looks like for each team members name) and so it would make more sense to use a class name, making the CSS easier to understand. It is better to use class names than to imply meaning from the nesting of selectors (i.e. this implicitly means/assumes that an li > span is a contributor name)
2. React automatically create spans in certain situations (when you do, for example `<div><img/>text</div>`, the "text" will get wrapped in a span), and so styling all spans may have unintended consequences because the selector is too broad (which also ties in with 1)